### DISCUSSION TITLE
Show entity number and brush number somewhere in the toolbar?

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Garux/netradiant-custom#190] [Ideas]
Would it be possible to add so you can see the selected entity and/or brush number in the toolbar, or somewhere else? When I open a map file as text I can see the IDs and I can search for these IDs too, but it would be nice to see this information in the editor too, not just in notepad.

**Comment by ensiform** [comment]
Are brush numbers stable in radiant before saving? I thought at least in old versions they move around, but maybe I'm only thinking of brush faces that are not/were not consistent when saved to map file.

**Comment by Hipshot-zz** [comment]
They should be stable and to my experience, they were stable in old radiants, at least 1.4. Actually, I don't think I have seen these change around IDs per save on the same brush ever in editors for games like these.

This is only from maybe a more personal convivence.

**Comment by Garux** [comment]
![image](https://github.com/Garux/netradiant-custom/assets/1734117/cd72f504-4651-484f-8c6b-66a48fc80d26)

**Reply by Hipshot-zz** [reply]
Ah, yes, but that's just to find a brush, that doesn't tell what the selected brush' ID or Entity number.

**Reply by Garux** [reply]
It says.

**Reply by Hipshot-zz** [reply]
Oh haha, right. yea it does, great! Thanks for pointing out.
